### PR TITLE
Add croatian language

### PR DESCRIPTION
**Pull request by Derkades**

### CODE REVIEW

**Comment by Aberdeener** [comment]
For my own understanding, why is this going to v2 instead of release/2.0.2?

**Comment by Derkades** [comment]
I thought everything should be going to the `v2` branch, and then if deemed important enough backported to existing release branches? Like a release branch for 2.0.x, 2.1.x, 2.2.x, 2.3.x, as discussed previously. I'm not sure why the release/2.0.2 branch exists.

**Comment by Aberdeener** [comment]
@samerton ?

**Comment by partydragen** [comment]
v2 was for releases only with is even why v2 branch is protected and requires pull request reviews

File: scripts/create_lang.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    modules = [e.name for e in Path('modules').iterdir() if e.name != 'Core']
+    paths = [Path('modules', module, 'language', name + '.json') for module in modules]
+    paths.append(Path('custom', 'languages', name + '.json'))
+
+    for path in paths:
+        with path.open('bw+') as f:
+            f.write(b'{}\n')
+
+    print("Done. Don't forget to add the language to core/classes/Language/Language.php")
```
Comment (Aberdeener) [inline scripts/create_lang.py:19]: Nice, this will be handy

**Review by Aberdeener — APPROVED** [review]

**Pull request merged**